README.md

```diff
@@ -91,7 +91,7 @@ Unofficial implementation of Fluent UI for [Flutter](flutter.dev). It's written
 
 ## Motivation
 
-Since flutter has Windows support (currently in stable under an early release flag as of 30/03/2021), it's necessary to have support to its UI guidelines to build apps with fidelity, since it has support for Material and Cupertino.
+Since Flutter has stable Windows support, it's necessary to have support to its UI guidelines to build apps with fidelity, since it has support for Material and Cupertino.
```

Comment:
```suggestion
Since Flutter has stable Windows support, it's necessary to have support to its UI guidelines to build apps with fidelity, the same way it has support for Material and Cupertino.
```